mungegithub/submit-queue/ingress.yaml

```diff
@@ -28,3 +31,10 @@ spec:
         - backend:
             serviceName: test-infra-sq-status
             servicePort: 80
```

Comment:
Done, though I don't have access to update DNS, so someone else will have to do that after this is deployed.